addons/web/static/src/fields/input_field_hook.js

```diff
@@ -23,6 +25,7 @@ export function useInputField(getValue, refName = "input") {
         lastSetValue = ev.target.value;
         isDirty = false;
     }
+    useBus(env.bus, "commit-change", setValue);
```

Comment:
For the main bus, we namespace the event names (+ uppercase), for instance "FIELD:COMMIT_CHANGE"

Reply:
useBus(env.bus, "commit-change", () => this.props.update(inputRef.el.value));